pkg/collector/corechecks/containers/generic/processor.go

```diff
@@ -160,7 +160,10 @@ func (p *Processor) processContainer(sender aggregator.Sender, tags []string, co
 		p.sendMetric(sender.Gauge, "container.memory.kernel", containerStats.Memory.KernelMemory, tags)
 		p.sendMetric(sender.Gauge, "container.memory.limit", containerStats.Memory.Limit, tags)
 		p.sendMetric(sender.Gauge, "container.memory.soft_limit", containerStats.Memory.Softlimit, tags)
-		p.sendMetric(sender.Gauge, "container.memory.rss", containerStats.Memory.RSS, tags)
+		// Filter out aberant values
+		if *containerStats.Memory.RSS < 1<<63 {
```

Comment:
I believe the check should be implemented in `pkg/util/cgroups` or in the `system` metrics collector `pkg/util/containers/metrics` as this is not the only where it could be needed.

Reply:
@vboulineau Is it better like this: https://github.com/DataDog/datadog-agent/pull/13824/files#diff-86b43eb3cbcd3f09bb1046d234a835f3acbea70abc41fdd6a453757bc532ee15 ?